spark/src/main/scala/com/alibaba/graphar/reader/EdgeReader.scala

```diff
@@ -323,8 +331,13 @@ class EdgeReader(prefix: String,  edgeInfo: EdgeInfo, adjListType: AdjListType.V
    */
   def readEdges(addIndex: Boolean = true): DataFrame = {
     val adjList_df = readAllAdjList(false)
-    val properties_df = readAllEdgePropertyGroups(false)
-    val df = DataFrameConcat.concat(adjList_df, properties_df)
+    val property_groups = edgeInfo.getPropertyGroups(adjListType)
+    val df = if (property_groups.size == 0) {
+      adjList_df
```

Comment:
NB: this is fixing the bug that when there are only adj list and no property group, the concatenated DataFrame in line 339 partition is not same between `adjList_df` and `properties_df`(the `properties_df` is a empty DataFrame and its partition num is 0).